itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,231 @@ object CheckCommand "printer_health" {
+			description = "Critical Threshold Battery max"
+		}
+		"--batt-current-critical-min" = {
+			value = "$thola_ups_batt_current_critical_min$"
+			description = "Critical Threshold Battery min"
+		}
+		"--batt-current-warning-max" = {
+			value = "$thola_ups_batt_current_warning_max$"
+			description = "Warning Threshold Battery max"
+		}
+		"--batt-current-warning-min" = {
+			value = "$thola_ups_batt_current_warning_min$"
+			description = "Warning Threshold Battery min"
+		}
+		"--batt-temperature-critical-max" = {
+			value = "$thola_ups_batt_temperature_critical_max$"
+			description = "Critical Threshold Battery Temperature max"
+		}
+		"--batt-temperature-critical-min" = {
+			value = "$thola_ups_batt_temperature_critical_min$"
+			description = "Critical Threshold Battery Temperature min"
+		}
+		"--batt-temperature-warning-max" = {
+			value = "$thola_ups_batt_temperature_warning_max$"
+			description = "Warning Threshold Battery Temperature max"
+		}
+		"--batt-temperature-warning-min" = {
+			value = "$thola_ups_batt_temperature_warning_min$"
+			description = "Warning Threshold Battery Temperature min"
+		}
+		"--current-load-critical-max" = {
+			value = "$thola_ups_current_load_critical_max$"
+			description = "Critical Threshold Current Load max"
+		}
+		"--current-load-critical-min" = {
+			value = "$thola_ups_current_load_critical_min$"
+ 			description = "Critical Threshold Current Load min"
+		}
+		"--current-load-warning-max" = {
+			value = "$thola_ups_current_load_warning_max$"
+			description = "Warning Threshold Current Load max"
+		}
+		"--current-load-warning-min" = {
+			value = "$thola_ups_current_load_warning_min$"
+			description = "Warning Threshold Current Load min"
+		}
+		"--rectifier-current-critical-max" = {
+			value = "$thola_ups_rectifier_current_critical_max$"
+			description = "Critical Threshold Rectifier Current max"
+		}
+		"--rectifier-current-critical-min" = {
+			value = "$thola_ups_rectifier_current_critical_min$"
+			description = "Critical Threshold Rectifier Current min"
+		}
+		"--rectifier-current-warning-max" = {
+			value = "$thola_ups_rectifier_current_warning_max$"
+			description = "Warning Threshold Rectifier Current max"
+		}
+		"--rectifier-current-warning-min" = {
+			value = "$thola_ups_rectifier_current_warning_min$"
+			description = "Warning Threshold Rectifier Current min"
+		}
+		"--system-voltage-critical-max" = {
+			value = "$thola_ups_system_voltage_critical_max$"
+			description = "Critical Threshold System Voltage max"
+		}
+		"--system-voltage-critical-min" = {
+			value = "$thola_ups_system_voltage_critical_min$"
+			description = "Critical Threshold System Voltage min"
+		}
+		"--system-voltage-warning-max" = {
+			value = "$thola_ups_system_voltage_warning_max$"
+			description = "Warning Threshold System Voltage max"
+		}
+		"--system-voltage-warning-min" = {
+			value = "$thola_ups_system_voltage_warning_min$"
+			description = "Warning Threshold System Voltage min"
```

Comment:
As the others these descriptions don’t have to be full sentences. But I'd like them to be more...